swift/ql/test/query-tests/Security/CWE-135/StringLengthConflation.swift

```diff
@@ -0,0 +1,125 @@
+
+// --- stubs ---
+
+func print(_ items: Any...) {}
+
+typealias unichar = UInt16
+
+class NSString
```

Comment:
We might want to declare a mockup of [NSObject](https://developer.apple.com/documentation/objectivec/nsobject) and make `NSString` extend that class to keep the types realistic.

Reply:
Will do.

Reply:
Done.